src/app/common/error-messages.ts

```diff
@@ -4,10 +4,13 @@ export enum SendFormErrorMessages {
   SameAddress = 'Cannot send to yourself',
   AmountRequired = 'You must specify an amount',
   MustNotBeZero = 'Must be more than zero',
+  MustBeNumber = 'Amount of STX must be a number',
   DoesNotSupportDecimals = 'This token does not support decimal places',
   InsufficientBalance = 'Insufficient balance. Available:',
   MustSelectAsset = 'You must select a valid token to transfer',
   TooMuchPrecision = '{token} can only have {decimals} decimals',
   MemoExceedsLimit = 'Memo must be less than 34-bytes',
   AdjustedFeeExceedsBalance = 'The fee added now exceeds your current STX balance.',
+  Only9Decimals = 'CAT can only have 9 decimals',
```

Comment:
This `enum` should be for generic errors only. This one specifies one specific to CAT token. Is this needed?

Reply:
Thaks Kyran will go back through and update the test.

Reply:
I can update this

Reply:
Still needs to be removed. This is for generic errors.

Reply:
ok I can remove this test but it does verify decimals

Reply:
You don't need to remove the test, my other suggestion (moving the string to the test), allows you to remove this line & keep the test